src/node_internals.h

```diff
@@ -245,7 +245,7 @@ class DebugSealHandleScope {
  public:
   explicit inline DebugSealHandleScope(v8::Isolate* isolate)
```

Comment:
Would it be possible add a default argument of `nullptr` here:
```c++
explicit inline DebugSealHandleScope(v8::Isolate* isolate = nullptr) 
```
And then be able to leave out the `nullptr` in the calls to the constructor?
```c++
DebugSealHandleScope handle_scope();
```

Reply:
@danbev Sure, done!